lib/pact/command/yaml_reader.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Kadena.Pact.Command.YamlReader do
+  @moduledoc """
+  Reads YAML file to create a command.
+  """
+
+  @type path :: String.t()
+  @type root_path :: String.t()
+  @type map_result :: map()
+  @type processed_map :: {:ok, map_result()} | {:error, Keyword.t() | struct()}
+
+  @spec read_yaml(path :: path()) :: processed_map()
+  def read_yaml(path) do
```

Comment:
maybe this function can be called just `read`, what do you think?

Reply:
Agree